card/mod/05_standard/spec/chunk/link_spec.rb

```diff
@@ -58,7 +58,7 @@ def assert_link target, args
     expect(content).to eq "wgw&nbsp; <a target=\"_blank\" class=\"external-link\" href=\"http://www.google.com\">google</a> &nbsp;  <br>"
   end
 
-  it "handles inclusions in link text" do
+  it "handles nests in link text" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.